### ISSUE TITLE
migration: indexing errors compilation

### ISSUE THREAD

**Issue by jalavik** [inspirehep/inspire-next#823]
Since the latest migration I note 115 errors related to indexing reported via mail alerts from Celery. 

One example:
```
Task inspirehep.modules.migrator.tasks.migrate_chunk 
with id c9af7532-7c6b-46fc-bf9e-848c08e16362 raised exception:
'BulkIndexError(u\'1 document(s) failed to index

.\', [{u\'index\': {u\'status\': 400, u\'_type\': u\'record\', 
u\'_id\': u\'962940\', u\'_index\': u\'jobshidden_v2\', 
u\'error\': {u\'type\': u\'mapper_parsing_exception\',
 u\'reason\': u\'failed to parse [deadline_date]\', 
u\'caused_by\': {u\'reason\': u\'Invalid format: "2011-01" is malformed 
at "-01"\', u\'type\': u\'illegal_argument_exception\'}}}}])'
```

Search your mailbox for `Task inspirehep.modules.migrator.tasks.migrate_chunk` on Feb 2-3.

**Comment by kaplun** [comment]
To me they all seem related to parsing the date. The current Conference mapping is too strict and expect only years.

CC: @ioannistsanaktsidis, @jmartinm

**Comment by kaplun** [comment]
This should have been fixed mostly be the new #842. @jalavik ?

**Comment by jalavik** [comment]
Indeed! I forgot the closing tag. Cheers.
